src/Components/ReflectionLibrary/ReflectionLibrary.test.js

```diff
@@ -1,3 +1,63 @@
-// It should display the loading message
-// It should display ReflectionCards after a successful fetch request
-// It should display an error message after an unsuccessful fetch request
+import { render, waitFor, screen } from '@testing-library/react';
+import ReflectionLibrary from './ReflectionLibrary';
+import { MemoryRouter } from 'react-router-dom';
+import { getReflections } from '../../apiCalls.js';
+import '@testing-library/jest-dom';
+
+jest.mock('../../apiCalls.js');
+
```

Comment:
Did we try jest.mock(getReflections) here?

Reply:
Not here, but we did in test itself and it was still buggy. But could have been a syntax mistake on our part.